external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,588 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+
+##### Pivoted single-measure data set
+
+| Year | Location  | Number of Arthur's Bakes | Status      |
+|:-----|:----------|-------------------------:|:------------|
+| 2022 | London    |                       35 | Provisional |
+| 2021 | Cardiff   |                       26 | Final       |
+| 2020 | Edinburgh |                       90 | Final       |
+| 2021 | Belfast   |                        0 | Final       |
+
+Notice that the observation value column title (`Number of Arthur's Bakes`) does not use one of the [reserved column names](../configuration/convention.md#conventional-column-names), and therefore the `type` field must be set to `observations` in order for csvcubed to recognise it as such. In the example below, the `label` field has been configured for both the `unit` and `measure` fields. Please refer to the [Measures Configuration](#measures-configuration) and [Units Configuration](#units-configuration) sections for details of further configuration options available:
+
+```json
+{ ...
+   "columns": {
+      "Number of Arthur's Bakes": {
+         "type": "observations",
+         "data_type": "integer",
+         "unit": {
+            "label": "Count"
+         },
+         "measure": {
+            "label": "Number of Stores"
+         }
+      }
+   }
+}
+```
+
+##### Pivoted multi-measure data set
+
+| Year | Location | Number of Arthur's Bakes | Revenue | Revenue Units  |
+|:-----|:---------|-------------------------:|--------:|:---------------|
+| 2022 | London   |                       35 |      25 | GBP (Sterling) |
+| 2021 | Cardiff  |                       26 |      18 | GBP (Sterling) |
+
+In this example, a second observation value column, `Revenue`,  has been added, with the associated unit contained in the `Revenue Units` column. As you can see, the `Revenue Units` column has been linked to the `Revenue` column through the `describes_observations` field. This must be formatted in exactly the same way for csvcubed to recognise the link and generate the correct results. However, there is no measure information associated for either of the observation value columns; this can be configured as follows:
+
+```json
+{ ...
+   "columns": {
+      "Number of Arthur's Bakes": {
+         "type": "observations",
+         "data_type": "integer",
+         "unit": {
+            "label": "Count"
+         },
+         "measure": {
+            "label": "Number of Stores"
+         }
+      },
+      "Revenue": {
+         "type": "observations",
+         "data_type": "decimal",
+         "measure": {
+            "label": "Revenue"
+         }
+      },
+      "Revenue Units": {
+         "type": "units",
+         "describes_observations": "Revenue"
+      }
+   }
+}
+```
+
+## Measures and Units (for the guide)
+
+Measures can either be attached to a Measure Column if there are multiple different measures in your data set, or to an Observation column's `measure` field if all observations in the cube have the same measure.
```

Comment:
It's more that all observations in the *column* have the same measure.

What's written here used to be true but isn't accurate now that we support pivoted multi-measure inputs.